v0.3/tutorials/quickstart clock.eve

```diff
@@ -0,0 +1,145 @@
+---
+layout: default
+title: Quickstart + Counter
+---
+
+{% raw %}
+
+# Quickstart + Counter
+
+This guide expands on the quickstart by adding a live clock to the `Clock` page.
+
+## Quickstart code
+
+This is made to add onto the web app you made in the quickstart. You'll want to add the new code from here into the quickstart.eve document if you've already got the web app written there. Otherwise, you can quickly copy and paste the code in this section to recreate it. Whichever path you take, jump down to the `Building the clock` section to begin working on the counter!
+
+Header
+
+````
+```
+commit
+  [#ui/div text: "My Quickstart App"]
+```
+````
+
+Layout
+
+````
+```
+commit
+  [#ui/div #qs-wrapper children:
+    [#ui/div #qs-logo]
+    [#ui/div #qs-nav-bar]
+    [#ui/div #qs-contents]]
+```
+````
+
+Pages for the web app
+
+````
+```
+commit
+  [#page name: "Home" sort: 1]
+  [#page name: "Counter" sort: 2]
+  [#page name: "Clock" sort: 3]
+```
+````
+
+Adding buttons to the Nav Bar
+
+````
+```
+search
+  page = [#page name sort]
+  nav-bar = [#ui/div #qs-nav-bar]
+
+commit
+  nav-bar.children += [#ui/div #qs-nav-btn page text: name sort]
+```
+````
+
+Starting Page
+
+````
+```
+commit
+  [#app page: "Home"]
+```
+````
+
+Home Page
+
+````
+```
+search
+  [#app page: "Home"]
+  view = [#qs-contents]
+
+bind
+  view.children += ([#ui/div text: "This is the Eve Quickstart app! Choose a link above to see some of the things Eve can do."], [#ui/img #qs-flappy src:"http://i.imgur.com/sp68LtM.gif"])
+```
+````
+
+Navigation
+
+````
+```
+search
+  click = [#html/event/click element:[#ui/div #qs-nav-btn page]]
+  view = [#app]
+
+commit
+  view.page := page.name
+```
+````
+
+## Building the clock
```

Comment:
Seems like this should be H2 (as it is) but the blocks beneath should be H3. They're all part of building the clock, right?